web/template/certificate_provider_start.gohtml

```diff
@@ -8,7 +8,7 @@
       {{ trHtml .App "beACertificateProviderContent" }}
 
       <p class="govuk-body">
-        <a href="{{ link .App .App.Paths.CertificateProvider.EnterReferenceNumber.Format }}" role="button" draggable="false" class="govuk-button govuk-button--start" data-module="govuk-button">
+        <a href="{{ link .App .App.Paths.CertificateProvider.Login.String }}" role="button" draggable="false" class="govuk-button govuk-button--start" data-module="govuk-button">
```

Comment:
can you use `.Format` in templates, it's a weak distinction for these paths but my thinking was that `.String` would only be used for registering and `.Format` would be used everywhere else, that way if their signature changes we'd get an error

Reply:
Sure - I wonder if there's any way to enforce this at the func level 🤔

Reply:
could write a little linter for the CI pipeline, but I think that would be overkill